src/activities/assignments/AssignmentEntity.js

```diff
@@ -894,6 +927,11 @@ export class AssignmentEntity extends Entity {
 			fields.push({ name: 'allowableFileType', value: assignment.allowableFileType });
 		}
 
+		const allowableFileTypeCustomValue = '5';
+		if (assignment.allowableFileType === allowableFileTypeCustomValue) {
```

Comment:
Should we check whether the `update-custom-allowable-file-type` action is present?
Should we check if `assignment.customAllowableFileTypes` is defined?

Reply:
Added!